airflow/cli/commands/task_command.py

```diff
@@ -450,13 +448,12 @@ def task_failed_deps(args):
 
     dep_context = DepContext(deps=SCHEDULER_QUEUED_DEPS)
     failed_deps = list(ti.get_failed_dep_statuses(dep_context=dep_context))
-    # TODO, Do we want to print or log this
     if failed_deps:
-        print("Task instance dependencies not met:")
```

Comment:
I had a look at some of the other `*_command.py` modules and many of them use `print(...)`. I wonder if there is some reason for this? Perhaps someone else from the community has the historical knowledge about that.

Cc. @eladkal @potiuk

Reply:
Personally those messages look like they should be logged, unless indeed there is another reason for them to be printed

Reply:
no reason that I'm aware of

Reply:
If we use logging here the output would be affected by custom logging config, but in these CLI parts we _should_ log to stdout (or stderr) regardless of how the user configures Airflow logging. Using `print` is correct for this purpose.

Reply:
Correct. All the command lines have `@suppress_logs_and_warnings` set so that the output is "clen" and does not contain extra stuff thet it should not be there. Particularly when `--output json` is used and eny extra print could for example break anythong that parses "json". So some of the print methods are deliberatly using print and not log because otherwise they would be suppressed. 

I think this change shoudl be changed in something different. It should review all such `print` calls and possibly replace them with another methos (`print_to_command_output`) and the command should have appropriate docstring explaining it's purpose.

Then all the "expected" prints should remain and use this dedicated function, and all the "accidental" ones should be turned into logs. 

This could avoid any future attempts of converting those prints and no further explanation and asking "elders" will be needed :)

Reply:
>I think this change shoudl be changed in something different. It should review all such print calls and possibly replace them with another methos (print_to_command_output) and the command should have appropriate docstring explaining it's purpose.

Love this :+1: